TrainingLoop/Callbacks/CSVLogger.swift

```diff
@@ -0,0 +1,70 @@
+import Foundation
+import ModelSupport
+
+/// A callback-based handler for logging the statistics to CSV file.
+public class CSVLogger {
+  public var path: String
+
+  let foundationFS: FoundationFileSystem
+  let foundationFile: FoundationFile
```

Comment:
Doc comment please. Don't name things after their types.

If you rename the thing `CSVLogFile` or something, then:

```suggestion
  let underlyingFile: FoundationFile
```

or

```suggestion
  let rawFile: FoundationFile
```

Reply:
Removed foundationFile.